.env

```diff
@@ -28,7 +28,7 @@ GOOGLE_APPLICATION_DECODE=
 
 
 # Frontend
-NEXT_PUBLIC_API_URL=https://dev.choco.saenyakorn.dev/api
-NEXT_PUBLIC_SOCKET_API_URL=https://dev.choco.saenyakorn.dev
+NEXT_PUBLIC_API_URL=http://localhost:3333/api
+NEXT_PUBLIC_SOCKET_API_URL=http://localhost:3333
 NEXT_PUBLIC_OMISE_PUBLIC_KEY=
```

Comment:
Not an accidental commit right? looks out of place with the rest of the PR changes

Reply:
No. The `.env` file is just a template of the environment. As you can see the setup instruction in `README.md`, you have to copy `.env` to `.env.local` to start the development app. So, the old `.env` misled the newbie developer to use `dev.choco.saenyakorn.dev` as a API domain. (Many people in our team faced the bug they can't solve since the bug was from `dev`, not `local` -> waste the time!!!)

Reply:
I see.